src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -136,8 +136,9 @@ public static class RecipeOptions {
         @Config.Comment({"Whether to make Glass related recipes harder. Default: true"})
         public boolean hardGlassRecipes = true;
 
-        @Config.Comment({"Whether to nerf Wood crafting to 2 Planks from 1 Log, and 2 Sticks from 2 Planks.", "Default: false"})
-        public boolean nerfWoodCrafting = false;
+        @Config.Comment({"Whether to nerf/Buff Wood crafting to X Planks from 1 Log, and X Sticks from 2 Planks.", "Default: 4"})
+        @Config.RangeInt(min = 0, max = 4)
```

Comment:
Do we really want to allow players to set value to 0? This will prevent them from making planks and stick at all. Sure this could be useful for modpack makers but they have other ways like CT.